doc/classes/AudioStreamPlayer.xml

```diff
@@ -44,6 +44,17 @@
 				Plays a sound from the beginning, or the given [param from_position] in seconds.
 			</description>
 		</method>
+		<method name="play_scheduled" experimental="">
+			<return type="void" />
+			<param index="0" name="absolute_time" type="float" />
+			<param index="1" name="from_position" type="float" default="0.0" />
+			<description>
+				Schedules a sound to be played on the [AudioServer]'s timeline at [param absolute_time] in seconds (see [method AudioServer.get_absolute_time]). If [param absolute_time] is in the past, the sound will be played immediately. The sound starts from the given [param from_position] in seconds.
```

Comment:
I actually prefer the prior description, but I understand the reasoning for the change. You wanted to make it more accessible and easier to imagine. Two problems though, as I understand them:
- `AudioServer.get_absolute_time` is essential for this method to be used
- The concept of "past" and "future" is a bit nebulous. Even `get_absolute_time`'s description states it's not quite accurate.
```suggestion
				Schedules a sound to be played on the [AudioServer]'s timeline at [param absolute_time] in seconds. If the sound is scheduled to play earlier than the value returned by [method AudioServer.get_absolute_time], it will be played immediately. The sound starts from the given [param from_position] in seconds.
```
or similar.